src/components/AffectExpandableForm.vue

```diff
@@ -44,9 +44,9 @@ const resolutionLabel = computed(() => {
   const resolution: string = props.affect.resolution || '';
   const resolutionValue = {
     [resolution]: resolution,
-    'DELEGATED': 'Delegated',
-    'WONTFIX': 'Won\'t Fix',
-    'OOSS': 'OOSS',
+    [affectResolutions.Delegated]: 'Delegated',
+    [affectResolutions.Defer]: 'Defer',
+    [affectResolutions.Ooss]: 'OOSS',
```

Comment:
I think we should change this, right now it works because we are overriding the `[resolution]: resolution` value.
Since this is only changing the **case** of the value (before we created the `won't fix` string), we could do:

```js
const resolutionValue = resolution.charAt(0) + resolution.slice(1).toLowerCase();
```